openedx/core/djangoapps/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -142,123 +230,179 @@ def setUp(self):
         )
 
         # OrgA and OrgB taxonomy
-        self.tC1 = Taxonomy.objects.create(name="tC1", enabled=True)
+        self.tBA1 = Taxonomy.objects.create(name="tBA1", enabled=True)
         TaxonomyOrg.objects.create(
-            taxonomy=self.tC1,
+            taxonomy=self.tBA1,
             org=self.orgA,
             rel_type=TaxonomyOrg.RelType.OWNER,
         )
         TaxonomyOrg.objects.create(
-            taxonomy=self.tC1,
+            taxonomy=self.tBA1,
             org=self.orgB,
             rel_type=TaxonomyOrg.RelType.OWNER,
         )
-        self.tC2 = Taxonomy.objects.create(name="tC2", enabled=False)
+        self.tBA2 = Taxonomy.objects.create(name="tBA2", enabled=False)
         TaxonomyOrg.objects.create(
-            taxonomy=self.tC2,
+            taxonomy=self.tBA2,
             org=self.orgA,
             rel_type=TaxonomyOrg.RelType.OWNER,
         )
         TaxonomyOrg.objects.create(
-            taxonomy=self.tC2,
+            taxonomy=self.tBA2,
             org=self.orgB,
             rel_type=TaxonomyOrg.RelType.OWNER,
         )
 
+    def setUp(self):
+
+        super().setUp()
+
+        self._setUp_orgs()
+        self._setUp_courses()
+        self._setUp_library()
+        self._setUp_users()
+        self._setUp_taxonomies()
+
 
 @skip_unless_cms
 @ddt.ddt
 @override_settings(FEATURES={"ENABLE_CREATOR_GROUP": True})
-class TestTaxonomyViewSet(TestTaxonomyObjectsMixin, APITestCase):
+class TestTaxonomyListCreateViewSet(TestTaxonomyObjectsMixin, APITestCase):
     """
-    Test cases for TaxonomyViewSet when ENABLE_CREATOR_GROUP is True
+    Test cases for TestTaxonomyReadViewSet when ENABLE_CREATOR_GROUP is True
```

Comment:
Does `ENABLE_CREATOR_GROUP` have any effect at all on the taxonomy code/permissions? I'm unclear why we have different tests based on that.

Reply:
I think it's confusing to mention that flag if it has no effect.

Reply:
Yeah I agree -- we used to care about this flag, but we don't anymore with the new permissions, so these extra tests can be removed.

Reply:
Great! I will fix this tomorrow ping you here!